File: fw-portal-web/src/main/resources/static/resources/js/dto/NotaDto.js
class NotaDto{
    _numero;
    _serie;
    _chave;

    constructor(numero,serie,chave){
        this._numero = numero;
        this._serie = serie;
        this._chave = chave;
    }

    getNumero(){
        return this._numero;
    }

    getSerie(){
        return this._serie;
    }
    getChave(){
        return this._chave;
    }
    
}

File: fw-portal-web/src/main/java/com/faraway/fwportal/dto/NotaDto.java
package com.faraway.fwportal.dto;

import com.faraway.fwportal.model.domain.Nota;
import com.fasterxml.jackson.annotation.JsonRootName;

@JsonRootName("nota")
public class NotaDto implements Comparable<NotaDto> {

	private String numero;

	private String serie;

	private String chave;

	public NotaDto(Nota nota) {
		super();
		this.numero = nota.getNumero();
		this.serie = nota.getSerie().strip();
		this.chave = nota.getChave().strip();
	}

	public String getNumero() {
		return numero;
	}

	public String getSerie() {
		return serie;
	}

	public String getChave() {
		return chave;
	}

	@Override
	public int compareTo(NotaDto o) {
		int byNumero = numero.compareTo(o.numero);
		int bySerie = serie.compareTo(o.serie);
		return byNumero - bySerie;
	}

}
